src/cephadm/cephadmlib/daemon_form.py

```diff
@@ -95,6 +95,26 @@ def __subclasshook__(cls, other: Type[DF]) -> bool:
         return callable(getattr(other, 'firewall_service_name', None))
 
 
+class UserGroupDaemonForm(DaemonForm, metaclass=abc.ABCMeta):
```

Comment:
rename "UserGroupIDDaemonForm" for clarity

Reply:
done